docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -10,113 +10,155 @@ You can push {es-sec} cases to these third-party systems:
-* *API token* ({jira} connectors only): The API token or password used
-to authenticate {jira} updates.
-* *API key ID* ({ibm-r} connectors only): The API key used to authenticate
-{ibm-r} updates.
-* *API key secret* ({ibm-r} connectors only): The API key secret used to
-authenticate {ibm-r} updates.
-* *API token* ({swimlane} connectors only): The {swimlane} API authentication token used for HTTP Basic authentication. This is the personal access token for your user role.
-
-. Choose the connector type (for {swimlane} connectors only):
-** *All*: You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if you don't set field mappings now, you'll be prompted to do so if you want to use the connector for a case or a rule. The prompts no longer display once you set up the required mappings.
-** *Alerts*: Provide an alert ID and rule name.
-** *Cases*: Provide a case ID, a case name, comments, and a description.
+. Choose the connector type ({swimlane} only):
++
+|===
+
+| *All* | You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if
+you don’t set field mappings now, you’ll be prompted to do so if you want to use the connector for a case or a rule. This
+is because the prompts no longer display once you set up the required mappings.
 
+| *Alerts* | Provide an alert ID and rule name.
+
+| *Cases* | Provide a case ID, a case name, comments, and a description.
+
+|===
++
 . Save the connector.
 
 TIP: To see how to connect {elastic-sec} to {jira}, watch the <<connect-security-to-jira, tutorial>> at the end of this topic.
 
+[float]
+[[mapped-case-fields]]
+=== Mapped case fields
+
 To represent an {es-sec} case in an external system, {es-sec} case fields are
 mapped as follows:
 
 NOTE: Data from mapped case fields can be pushed to external systems but cannot be pulled in.
 
 * For {sn} incidents:
-** *Title*: Mapped to the {sn} `Short description` field. When an update to a
-Security case title is sent to {sn}, the existing {sn} `Short description`
-field is overwritten.
-** *Description*: Mapped to the {sn} `Description` field. When an update to a
-Security case description is sent to {sn}, the existing {sn} `Description`
-field is overwritten.
-** *Comments*: Mapped to the {sn} `Work Notes` field. When a comment is updated
-in a Security case, a new comment is added to the {sn} incident.
++
+|===
+
+| *Title* | Mapped to the {sn} `Short description` field. When an update to a case title is sent to {sn}, the existing {sn} `Short description` field is overwritten.
+
+| *Description* | Mapped to the {sn} `Description` field. When an update to a case description is sent to {sn}, the existing {sn} `Description` field is overwritten.
+
+| *Comments* | Mapped to the {sn} `Work Notes` field. When a comment is updated in a case, a new comment is added to the {sn} incident.
```

Comment:
There's a lot of repetition in this and the next three tables, so the more important info (A is mapped to B) gets lost in a lot of text. 
 
Would it be possible to add a column for "Mapped field" for all of these tables? That would relieve some of the repetition of "Mapped to the XXX `yyy` field" in every line, and it'd be instantly clear what is mapped to what. I know our narrow page margins make tables difficult, but maybe just one more column could still work...?